core/src/components/searchbar/searchbar.ionic.scss

```diff
@@ -9,16 +9,18 @@
    * @prop --focus-ring-color: The color of the ring around the focused element.
    * @prop --focus-ring-width: The width of the ring around the focused element.
    */
-  --background: #{globals.$ion-primitives-neutral-100};
+  --background: #{globals.$ion-bg-neutral-subtle-default};
```

Comment:
Design shows this to be `bg/input/bold/default`. Were you able to with the design team of which token to use?